tests/integration/routes/account/put-account-test.js

```diff
@@ -81,7 +81,7 @@ getServer(function (error, server) {
         .reply(201, [{
           id: 'org.couchdb.user:pat-doe',
           error: 'conflict',
-          reason: 'Document update conflict'
+          reason: 'An account with that username already exists'
```

Comment:
You don’t need to change this part. Here we simulate what CouchDD would return when the document already exists, and CouchDB returns "Document update conflict". See this for comparison: https://wiki.apache.org/couchdb/HTTP_Document_API#line-233